v2/projects/sablier/index.js

```diff
@@ -0,0 +1,31 @@
+const axios = require("axios");
+
+module.exports = {
+  /* Metadata */
+  name: "Sablier",
+  category: "Payments",
+  start: 1573582731,
+  token: null,
+  tokenHolderMap: [
+    {
+      tokens: async () => {
+        const result = await axios({
+          url: "https://api.thegraph.com/subgraphs/name/sablierhq/sablier",
```

Comment:
We prefer to avoid adding any 3rd party api call inside adapter. Please Use our SDK apis to pull on-chain data.

Reply:
It's not possible to not use a third-party API. The data "what tokens have been streamed on Sablier" cannot be obtained by querying a smart contract.